### ISSUE TITLE
Using 'mbtn_back' in 'input.conf' generates a warning on click

### ISSUE THREAD

**Issue by FreeTheTech101** [mpv-player/mpv#12768]
### Important Information

Provide following Information:
- mpv version: 20231028 - Nightly Build
- Platform and Version: Windows 11, 22H2
- Source of the mpv binary: shinchiro

### Reproduction steps

1. Add `mbtn_back seek -5 exact` to `input.conf`
2. Seek backwards using the appropriate button 
3. Check the MPV console or log output

![MPV](https://github.com/mpv-player/mpv/assets/7614507/e8cde2ac-3126-497e-9813-2f24edaf3a31)

### Expected behavior

Using the `mbtn_back` button should not generate a warning to the log file.

### Actual behavior

Using the `mbtn_back` button generates the warning "No key binding found for key 'BACK'." `mbtn_forward` works as intended, and adding `back seek -5 exact` to `input.conf` resolves the issue.

### Log file

[Log Download](https://github.com/mpv-player/mpv/files/13195718/Log.txt)

**Comment by zhongfly** [comment]
From the output, you pressed the key that is actually `BACK` and not `MBTN_BACK`
Maybe related to https://github.com/mpv-player/mpv/commit/fa378249dd79296d7b10542a7545d05d4b6d0844 , this commit add new keys `BACK`.

**Comment by FreeTheTech101** [comment]
I grabbed the last release since it occurred before those commits, and the warning is not being printed, so it does seem likely that one of those commits are the culprit. Just to confirm, I went ahead and commented out my config lines to see if the warning occurs with the default behaviour (cycling forwards and backwards through files), and it does. I sadly don't have my Linux environment setup to test this, so I can't say if it's an issue there.
